### DISCUSSION TITLE
Attach comment to API activity

### DISCUSSION THREAD

**Discussion by wfoxall** [directus/directus#17347] [Feature Requests]
A few times, I've wished I could include a comment with item update API requests. Sometimes as a way for a user to add a description of why they're updating an item, or perhaps some context when the update is triggered programmatically.
These could then be retrieved by retrieving activity filtered to the item(s) you're interested in.

The `directus_activity` table has a `comment` column which stores the text for comments added via the app. And the `directus_revisions` table has a foreign key to the activity table, so it seems feasible that this could be implemented quite neatly.
When using full accountability settings for a collection, it could even show up alongside the relevant revision in the Revisions panel in the app.

I've considered implement this myself for my own project(s), perhaps with a new "item_update" collection and M2A field. But it feels like quite a lot of the building blocks are already built into Directus, so thought I'd raise it as an idea here.

It seems like it might complement #2975 as a means of describing a future revision.

**Comment by rijkvanzanten** [comment]
😄  From my own notes on the future revisions feature:

<img width="451" alt="CleanShot 2023-01-30 at 16 01 44@2x" src="https://user-images.githubusercontent.com/9141017/215594101-331aeb22-fbbf-476b-8ffb-a98d3a36b8d3.png">

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
